aggregate.ts

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import path from 'path';
+import { reportToRow, reportToRowHeaders } from './report-to-row';
+import csvStringify from 'csv-stringify/lib/sync';
+
+const { readdir, writeFile } = fs.promises;
+
+/** Combines the individual report CSV's from a folder into a single CSV file */
+export const aggregateCSVReports = async (dataDirPath: string) => {
+  const reportsDirPath = path.join(dataDirPath, 'reports');
+  const files = await readdir(reportsDirPath);
+
+  const rows = [];
+  let headers: string[] | null = null;
+
+  files.forEach((fileName) => {
+    if (fileName !== '.DS_Store') {
+      const filePath = path.join(reportsDirPath, fileName);
+      const fileContents = fs.readFileSync(filePath, 'utf8');
+      // If headers aren't set yet, do it now
+      if (!headers) headers = reportToRowHeaders(fileContents);
+      console.log(`Bundling ${fileName} into aggregated report`);
```

Comment:
@emersonthis I was wondering if I could remove this console.log. It seems like it clutters the output a little bit

Reply:
sure

Reply:
Let's just make sure there's some other output that makes it clear bundling has started because on big sites this can take some time. This might already be in place. I can't remember.